dbt/task/compile.py

```diff
@@ -6,6 +6,8 @@
 import dbt.project
 from collections import defaultdict
 
+from ..compilation import Linker
```

Comment:
yeah, i put the Linker code in a file called `compilation.py` in the dbt root dir. Since it's up once directory, you gotta tell the interpreter where to look